src/main/java/com/amplitude/Amplitude.java

```diff
@@ -17,6 +19,11 @@
   private HttpCall httpCall;
   private HttpTransport httpTransport;
 
+  /**
+   * A dictionary of key-value pairs that represent additional instructions for server save operation.
+   */
+  private JSONObject options;
```

Comment:
```suggestion
  private Options options;
```

Would prefer this to be strongly typed similar to [Event](https://github.com/amplitude/Amplitude-Java/blob/main/src/main/java/com/amplitude/Event.java).

```
// Options.java
public class Options {
  /** A unique identifier for your event. */
  public String minIdLength;

 public JSONObject toJsonObject() { ... } => { "min_id_length": minIdLength };
}
```